checks/normandy/uptake_error_rate.py

```diff
@@ -20,15 +20,17 @@
 
 # Normandy uses the Uptake telemetry statuses in a specific way.
 # See https://searchfox.org/mozilla-central/rev/4218cb868d8deed13e902718ba2595d85e12b86b/toolkit/components/normandy/lib/Uptake.jsm#23-43
-NORMANDY_STATUSES = {
-    "custom_1_error": "recipe_action_disabled",
-    "backoff": "recipe_didnt_match_filter",
-    "apply_error": "recipe_execution_error",
-    "content_error": "recipe_filter_broken",
-    "download_error": "recipe_invalid_action",
-    "signature_error": "runner_invalid_signature",
+UPTAKE_STATUSES = {
+    "recipe_action_disabled": "custom_1_error",
+    "recipe_didnt_match_filter": "backoff",
+    "recipe_execution_error": "apply_error",
+    "recipe_filter_broken": "content_error",
+    "recipe_invalid_action": "download_error",
+    "runner_invalid_signature": "signature_error",
+    "action_pre_execution_error": "custom_1_error",
+    "action_post_execution_error": "custom_2_error",
 }
-UPTAKE_STATUSES = {v: k for k, v in NORMANDY_STATUSES.items()}
+NORMANDY_STATUSES = {v: k for k, v in UPTAKE_STATUSES.items()}
```

Comment:
`custom_1_error` appears as a value twice, is that OK?

Reply:
Ok I fixed it by joining with the source type (`recipe`, `action`, `runner`...)